src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/AddImports/IAddImportsService.cs

```diff
@@ -22,10 +22,17 @@ namespace Microsoft.CodeAnalysis.AddImport
 {
     [DataContract]
     internal record struct AddImportPlacementOptions(
-        [property: DataMember(Order = 0)] bool PlaceSystemNamespaceFirst,
-        [property: DataMember(Order = 1)] bool PlaceImportsInsideNamespaces,
-        [property: DataMember(Order = 2)] bool AllowInHiddenRegions)
+        [property: DataMember(Order = 0)] bool PlaceSystemNamespaceFirst = true,
+        [property: DataMember(Order = 1)] bool PlaceImportsInsideNamespaces = false,
+        [property: DataMember(Order = 2)] bool AllowInHiddenRegions = false)
     {
+        public AddImportPlacementOptions()
+            : this(PlaceSystemNamespaceFirst: true)
```

Comment:
structs suck.

Reply:
I'll keep it this way for now. We can switch all options struct to classes later on.